development/roles.md

```diff
@@ -252,12 +252,12 @@ Trino project. They are often associated to a specific aspect of running the
 project or a technology stack and integration for Trino.
 
 Examples for subproject repositories are the
-[trino.io]({{site.github_repo_url}}/grafana-trino) website source or the
-[grafana-trino]({{site.github_repo_url}}/grafana-trino) repository for the Trino
+[trino.io]({{site.github_org_url}}/grafana-trino) website source or the
```

Comment:
Found another issue... 

```suggestion
[trino.io]({{site.github_org_url}}/trino.io) website source or the
```

Reply:
oops didn't see that.
i applied the suggested fix, squashed and updated commit message. PTAL